adminsortable/admin.py

```diff
@@ -297,8 +297,11 @@ def do_sorting_view(self, request, model_type_id=None):
 
                 for index in indexes:
                     obj = objects_dict.get(index)
-                    setattr(obj, order_field_name, start_index)
-                    obj.save()
+                    # perform the update iff the order field has changed

```

Comment:
Thanks for the reply, I actually did mean iff: https://en.wikipedia.org/wiki/If_and_only_if.  But if you think it reads better as just "if" then I can definitely change it.  More importantly though, are you ok with the code?